deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```diff
@@ -143,3 +144,11 @@ spec:
         namespace: openshift-monitoring
         managed_notification_template: WorkerNodeFilesystemAlmostOutOfFiles
         send_managed_notification: "true"
+    - alert: KubeNodeUnschedulableSRE
+      # KubeNodeUnschedulableSRE alert firing).
+      expr: (kube_node_spec_unschedulable > 0 and on(node) kube_node_role{role="master"}) or (kube_node_spec_unschedulable > 0 unless ignoring(node,container,endpoint,job,namespace,service) sum(mapi_machine_set_status_replicas{name=~".*upgrade$"}) > 0)
+      labels:
+        severity: Info
+        namespace: openshift-monitoring
+        send_managed_notification: "true"
+        managed_notification_template: "KubeNodeUnschedulableSRE"    
```

Comment:
Is there a reason that we're redefining this here? It's already defined at https://github.com/openshift/managed-cluster-config/pull/2024/files#diff-4ec82a4ca9332ac099cc824aaf84ad8c9b48c6736af18aebcfea99e5644bf658L13, so we should be able to just update the labels there.

Reply:
Good point. I guess I did not fully understand how the alert directories are structured.